eng/common/pipelines/templates/steps/update-docsms-metadata.yml

```
@@ -76,12 +79,25 @@ steps:
 
 - pwsh: |
     $packageInfoJson = '${{ convertToJson(parameters.PackageInfoLocations) }}'.Trim('"')
```

```suggestion
    $packageInfoJson = '${{ convertToJson(parameters.PackageInfoLocations) }}'.Trim('"').Replace("\\", "/")
```

While in here can you please add this Replace which was an issue that @danieljurek hit while running on windows.